CMakeLists.txt

```diff
@@ -98,11 +99,21 @@ add_library(FAudio
 	src/FAudio_internal_simd.c
 	src/FAudio_operationset.c
 	src/FAudio_platform_sdl2.c
+	src/FAudio_platform_win32.c
 	# Optional source files
 	src/XNA_Song.c
 	src/FAudio_gstreamer.c
 )
 
+if(PLATFORM_WIN32)
+	target_link_libraries(FAudio PRIVATE -ldxguid -luuid -lwinmm -lole32 -ladvapi32 -luser32 -lmfplat -lmfreadwrite -lmfuuid -lpropsys)
+	target_compile_definitions(FAudio PUBLIC FAUDIO_WIN32_PLATFORM)
+	target_compile_definitions(FAudio PRIVATE HAVE_GSTREAMER=1)
```

Comment:
Slightly conflicts with what happens in gstreamer.c

Reply:
Probably some commit split leftovers, I'll make sure they make more sense individually.

BTW I kept the name GStreamer, but as it's later implemented on top of MF, it's a bit awkward, it may be worth renaming the related code to something like WMADEC, and enable it when either GStreamer is found, or on Win32 platform.

Reply:
The WMADEC option makes sense to me - for Win32 it can be built-in while for other targets it'll require GStreamer. We _might_ need to keep the `GSTREAMER` CMake option for compatibility, but from what I've seen packagers have been really good about changes to the build system since our changesets are so small anyway, so it may not be a huge deal.

Reply:
I kept the `GSTREAMER` CMake flag as it makes sense if someone wants to check for GStreamer availability, and eventual WMADEC support. The win32 platform has it enabled by default.